python/ray/_private/worker.py

```diff
@@ -2320,6 +2321,7 @@ def connect(
             finshes launching. If the worker is not launched by raylet (e.g.,
             driver), this must be -1 (default value).
         debug_source: Source information for `CoreWorker`, used for debugging and informational purpose, rather than functional purpose.
+        enable_resource_isolation: If true, core worker enables resource isolation through cgroupv2 by reserving resources for ray system processes.
```

Comment:
```suggestion
        enable_resource_isolation: If true, core worker enables resource isolation by adding itself into appropriate cgroup.
```

Do you plan on passing the cgroup path for workers as another argument? If so, we should add a TODO and update the comment with the variable name once we've created it.

Reply:
Yes, I left a TODO so after your PR gets merged, we could unify the naming and comments.